src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -486,5 +495,20 @@ private void setArchitectureAndWeightsFromResources() {
         }
     }
 
+    private void startPythonSession(){
```

Comment:
Can you rename this to something more specific - maybe one suggestion would be initializePythonArgsAndModel.

Reply:
I think it would be a lot simple if all of the code that figures out the values for `weights` and `architecture` was in one place, i.e., all of the if blocks in `setArchitectureAndWeightsFromResources` and `startPythonSession` together in one place. The case where you use Python `None` would require special treatment, but it would be much easier to read if the logic were consolidated.

Reply:
renamed and consolidated the fxns.